spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -1631,5 +1631,25 @@ def singleton_class(klass); end
         RUBY
       )
     end
+
+    it("sanitize parameter names creating through meta-programming") do
+      expect(
+        compile(template(<<~RUBY))
+          class Foo
+          <% if ruby_version(">= 2.7.0") %>
+            module_eval("def foo(...); end")
```

Comment:
Only for Ruby 2.7 as it will raise an exception at runtime for previous versions.